### ISSUE TITLE
[a11y] 9.1.1.1a - Alternative texts for control elements

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5375]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------
## General notes
The alternative text for the logo "Zur Seite "alle Dateien" navigieren" should contain the text"Logo own cloud.“

## Status: rather fulfilled

**Comment by pascalwengerter** [comment]
Superseded by the more recent #10698
